include/swift/AST/DiagnosticsDriver.def

```diff
@@ -114,6 +114,10 @@ ERROR(error_sdk_too_old,none,
 ERROR(error_ios_maximum_deployment_32,none,
       "iOS %0 does not support 32-bit programs", (unsigned))
 
+WARNING(warn_arclite_not_found_when_link_objc_runtime,none,
+        "Unable to find Objective-C runtime support library 'arclite'. "
+        "consider '-no-link-objc-runtime' flag", ())
```

Comment:
There are some formatting issues with this error message:

* "Unable" should be lowercase
* Use a semicolon instead of a period
* The phrasing of the last part is a little awkward; I might say something like `pass '-no-link-objc-runtime' to silence this warning`

Reply:
Sure :))

Reply:
I do see that "pass XYZ" sounds like a command, not a potential choice, but this phrase in compiler diagnostics are intended to read as "pass XYZ *if you want to* silence this warning". Maybe we should invert these in the future ("to silence this warning, pass XYZ"), but that makes it harder to see what you have to do. For now, let's stay consistent with the phrasing of other diagnostics.

Reply:
Just updated :)